packages/atlas-tokens/package.json

```diff
@@ -2,7 +2,7 @@
   "author": "ProductPlan <dev@productplan.com>",
   "name": "@productplan/atlas-tokens",
   "homepage": "https://github.com/ProductPlan/atlas",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I have ruined everything, and 0.2.0 was already published. this'll need a version bump